src/core/ConfigManager.cpp

```diff
@@ -282,7 +282,8 @@ void ConfigManager::addRecentlyOpenedProject( const QString & file )
 {
 	QFileInfo recentFile( file );
 	if( recentFile.suffix().toLower() == "mmp" ||
-			recentFile.suffix().toLower() == "mmpz" )
+			recentFile.suffix().toLower() == "mmpz" ||
+				recentFile.suffix().toLower() == "mpt" )
```

Comment:
You could remove the one tab of indentation here so these two lines are at the same level